internal/migrations/sql/019_add_detail_reservation_instances.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE reservation_instances ADD COLUMN
+  detail JSONB NOT NULL DEFAULT '{}'::jsonb
```

Comment:
We do just `detail JSONB` which is nullable with no default value in the other table but this is fine.